lib/r10k/forge/module_release.rb

```diff
@@ -174,21 +184,21 @@ def unpack(target_dir)
 
       # Remove all files created while downloading and unpacking the module.
       def cleanup
-        cleanup_unpack_path
-        cleanup_download_path
+        cleanup_unpack_root
+        cleanup_download_root
       end
 
       # Remove the temporary directory used for unpacking the module.
-      def cleanup_unpack_path
-        if unpack_path.exist?
-          unpack_path.rmtree
+      def cleanup_unpack_root
+        if unpack_root.exist?
```

Comment:
This seems like it'd be simpler as `File.dirname(unpack_path)`. I don't particularly like adding accessors for the parent directory when it's supposed to have a fixed relationship with `unpack_path`.

Same with `download_path`.